src/graph.ts

```diff
@@ -16,3 +16,11 @@ export interface INodeVariable {
   description?: string;
   enum?: string[];
 }
+
+export interface INodeExample {
+  key: string;
+  summary?: string;
+  description?: string;
+  value?: any;
```

Comment:
Does an Example with no value make sense? What's its purpose?

Reply:
It's because of the `externalValue` property, can have `value` or `externalValue`. Could create two separate interfaces, one with `value` required and one with `externalValue` required?

Reply:
What's the difference between a value and an externalValue? Can't they be unified in an unique value? Maybe with a `external: boolean` property? Or can they both exist together?

Reply:
They can't both exist together: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#example-object